src/Reports/SiteSummary.php

```diff
@@ -49,4 +60,35 @@ public function getReportField()
             ]);
         return $grid;
     }
+
+    /**
+     * @return string
+     */
+    protected function resolveCmsVersion()
+    {
+        $versionModules = ArrayLib::flatten($this->extend('getVersionModules'));
+        if (empty($versionModules)) {
+            $versionModules = array(
+                'silverstripe/framework' => 'Framework',
+                'silverstripe/cms' => 'CMS',
+            );
+        }
```

Comment:
I think it'd be good to provide this array to extensions, rather than define it as a fallback, e.g.:

```php
$versionModules = array(...);
$this->extend('updateVersionModules', $versionModules);
...
```

We could expect extensions to modify the array by reference